app/packs/src/apps/mydb/elements/details/wellplates/designerTab/WellContainer.js

```diff
@@ -53,41 +53,21 @@ class WellContainer extends Component {
   }
 
   render() {
-    const { style, isDragging, connectDragSource, connectDropTarget, well, isOver, canDrop, active } = this.props;
-    const containerStyle = {
-      paddingTop: 9,
-      borderRadius: '50%',
-      float: 'left',
-      borderWidth: 4,
-      borderStyle: 'solid',
-      borderColor: 'lightgray',
-      textAlign: 'center',
-      verticalAlign: 'middle',
-      lineHeight: 2,
-      cursor: 'move',
-      backgroundColor: well.color_code || 'white'
+    const { isDragging, connectDragSource, connectDropTarget, well, isOver, canDrop, active } = this.props;
+    const styleOverrides = {
+      backgroundColor: well.color_code || 'white',
     };
-    if (active) {
-      containerStyle.backgroundColor = '#337ab7';
-      containerStyle.color = 'white';
-    }
-    if (isDragging) {
-      containerStyle.opacity = 0;
-    }
-    if (well.sample) {
-      containerStyle.borderColor = 'gray';
-    }
-    if (isOver && canDrop) {
-      containerStyle.borderStyle = 'dashed';
-      containerStyle.borderColor = '#337ab7';
-      containerStyle.opacity = 0;
-    } else if (canDrop) {
-      containerStyle.borderStyle = 'dashed';
-    }
+
+    const classes = ['wellplate-well-container']
+    if (active) classes.push('active')
+    if (isDragging) classes.push('is-dragging')
+    if (well.sample) classes.push('has-sample')
+    if (canDrop) classes.push('can-drop')
+    if (isOver) classes.push('is-over')
```

Comment:
What do you mean? Utility in the sense of "can be used again for other dropzone related stuff"?

Reply:
Oh, I mean instead of defining classes like `.is-dragging` you could use a combination bootstrap utility classes like `opacity-25 border-dashed`.

Reply:
I checked and while it would be possible, I think for now we can just leave it at that. The colors will be changed soon anyway. The only thing we have a matching utility class for is border-style dashed, which I used